### PR TITLE
:sparkles: feat: added registration_ok screen

### PR DESCRIPTION
**Pull request by hartmannjean**
## Task

This pull request refers to task LOR 125

- [ ] Does not apply to this pull request

Added registration_ok screen

Please include a summary of the change and which issue is fixed.

## Types of changes #

- [x] New feature (non-breaking change which adds functionality).
- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## What changed: #

- [X] UI
- [X] Business rule
- [ ] Documentation
- [ ] Gradle or Build
- [X] Navigation
- [ ] Tests
- [X] Resource files
- [ ] Project architecture
- [ ] Code style

## Checklist #

- [X] Have tested the changes.
- [X] Met all the acceptance requirements of this task.
- [X] Verified if branch is up-to-date with `development` (if not - rebase it or merge it).
- [X] Resolve git conflicts

## If any of the acceptance criteria are not met, please explain why below #
What is the criteria? Why it is different?

### CODE REVIEW

File: app/src/main/java/com/loryblu/loryblu/navigation/NavGraph.kt [hunk 1]
```diff
@@ -46,7 +47,7 @@ fun SetupNavGraph(startDestination: String, navController: NavHostController) {
         registerChildRoute(
             navigateToHomeScreen = {
```
Comment (OdisBy) [inline app/src/main/java/com/loryblu/loryblu/navigation/NavGraph.kt]: Mude o nome dessa variável para corresponder a ação

File: app/src/main/java/com/loryblu/loryblu/navigation/Screen.kt [hunk 2]
```diff
@@ -7,4 +7,5 @@ sealed class Screen(val route: String) {
     object CreatePassword: Screen(route = "create_password_screen")
     object ForgetPassword: Screen(route = "forget_password_screen")
     object Home: Screen(route = "home_screen")
+    object RegistrationConfirmed: Screen(route = "registration_confirmed")
```
Comment (OdisBy) [inline app/src/main/java/com/loryblu/loryblu/navigation/Screen.kt]: Poem o padrão do "_screen" no final

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        }
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally
+
+        ){
+            Text(
+                fontSize = 20.sp,
+                text = "Cadastro efetuado com sucesso!",
```
Comment (OdisBy) [inline app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt]: A pedido do André: "Ponha as strings no strings.xml"

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                color = Color.Black,
+            )
+            Image(
+                modifier = Modifier.padding(top = 70.dp),
+                painter = painterResource(id = R.drawable.ic_ok_registration),
+                contentDescription = "image description",
```
Comment (OdisBy) [inline app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt]: hardcoded string

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt
Comment (OdisBy) [inline app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt:1]: A pedido do André: tirar o showToast ao criar a conta (não é nesse arquivo, crio que é na tela de cadastro da criança)

**Review by OdisBy — CHANGES_REQUESTED** [review]
Apenas para facilitar no que é para mudar que falamos ontem

File: app/src/main/java/com/loryblu/loryblu/navigation/NavGraph.kt (hunk 1 above)
Comment (hartmannjean) [reply app/src/main/java/com/loryblu/loryblu/navigation/NavGraph.kt]: Ok!

File: app/src/main/java/com/loryblu/loryblu/navigation/Screen.kt (hunk 2 above)
Comment (hartmannjean) [reply app/src/main/java/com/loryblu/loryblu/navigation/Screen.kt]: Ok!

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt (hunk 3 above)
Comment (hartmannjean) [reply app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt]: Ok!

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt (hunk 4 above)
Comment (hartmannjean) [reply app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt]: Ok!

File: app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt
Comment (hartmannjean) [reply app/src/main/java/com/loryblu/loryblu/register/registration/RegistrationConfirmedScreen.kt:1]: Ok!

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -32,6 +33,7 @@
     <string name="birthday_icon">Ícone de data de nascimento</string>
     <string name="boy_icon">Ícone do botão menino</string>
     <string name="girl_icon">Ícone do botão da menina</string>
+    <string name="registration_ok_icon">registration completed</string>
```
Comment (OdisBy) [inline app/src/main/res/values-pt-rBR/strings.xml]: Muita cara de pau

**Review by OdisBy — CHANGES_REQUESTED** [review]
omg the control c + control v request changes

**Review by OdisBy — APPROVED** [review]
Congratulations ㊗️

**Review by OdisBy — APPROVED** [review]
Inimigo do meu approve 👍

**Review by softdevandre — APPROVED** [review]
o approve da amizade, o approve dos brothers 🤝🫂

**Pull request merged**